crates/ruff/src/rules/pycodestyle/rules/logical_lines/space_around_operator.rs

```diff
@@ -120,6 +120,57 @@ impl Violation for MultipleSpacesAfterOperator {
     }
 }
 
+/// ## What it does
+/// Checks for extraneous tabs after a comma.
+///
+/// ## Why is this bad?
+/// Commas should be followed by one space, never tabs.
+///
+/// ## Example
+/// ```python
+/// a = 4,\t5
+/// ```
+///
+/// Use instead:
+/// ```python
+/// a = 4, 3
+/// ```
+///
+#[violation]
+pub struct TabAfterComma;
+
+impl Violation for TabAfterComma {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("Tab after comma")
+    }
+}
+
+/// ## What it does
+/// Checks for extraneous whitespace after a comma,
+///
+/// ## Why is this bad?
+/// According to the `black` code style, commas should be followed by a single space.
+///
+/// ## Example
+/// ```python
+/// a = 4,  5
```

Comment:
nit: makes the example a little easier to stop
```suggestion
/// a = 4,      5
```